config/application.rb

```diff
@@ -59,8 +59,15 @@ class Application < Rails::Application
     # MySQL bug where `rake db:reset` causes ActiveRecord to be loaded
     # before initializers and causes configuration not to be respected.
     config.load_defaults 6.0
+    # config.load_defaults 6.1
```

Comment:
Well, yes, the point is: `has_many_inversing` will probably stay for a while, most likely until Rails 7 or further, because there are still some open issues about it, so I wouldn't bother adjusting our code for something that still has an inconsistent behavior.

On the other hand, there are two switches in `new_framework_defaults_6_1.rb`, that are labeled with `NOTE: will be applied later`, so they are not applied yet. The reason being, the comments on them say:
```
# This change is not backwards compatible with earlier Rails versions.
# It's best enabled when your entire app is migrated and stable on 6.1.
```

But I agree that probably we can just override them here, with a comment, and load 6.1 defaults.